contracts/ConsensusUtils.sol

```diff
@@ -188,14 +206,6 @@ contract ConsensusUtils is EternalStorage, ValidatorSet {
     return addressArrayStorage[keccak256(abi.encodePacked("snapshot", _snapshotId, "addresses"))];
   }
 
-  function _setSnapshotStakeAmountForAddress(uint256 _snapshotId, address _address) internal {
-    uintStorage[keccak256(abi.encodePacked("snapshot", _snapshotId, "address", _address, "stakeAmount"))] = stakeAmount(_address);
-  }
-
-  function _getSnapshotStakeAmountForAddress(uint256 _snapshotId, address _address) internal view returns(uint256) {
-    return uintStorage[keccak256(abi.encodePacked("snapshot", _snapshotId, "address", _address, "stakeAmount"))];
-  }
-
```

Comment:
why did you removed this, was those functions ever been used?
are stake should be exactly 100K?

Reply:
Never used, probably leftovers from a long time ago...
Stake should be exactly 100k, can't stake more